Postgres Error during Install

This happened on Craft 5.0.3, PHP 8.3, Postgres 16 on DDEV

```
Copy Stacktrace Search Stackoverflow Search Google Exception
Migration Error – craft\errors\MigrationException

An error occurred while executing the "statikbe\carbontracker\migrations\Install migration: SQLSTATE[42804]: Datatype mismatch: 7 ERROR: column "green" is of type boolean but default expression is of type integer
HINT: You will need to rewrite or cast the expression.
The SQL being executed was: CREATE TABLE "carbontracker_stats" (
"id" serial NOT NULL PRIMARY KEY,
"entryId" integer NOT NULL,
"siteId" integer NOT NULL,
"url" varchar(255) NOT NULL,
"green" boolean DEFAULT 0,
"bytes" integer,
"cleanerThan" double precision DEFAULT 0,
"rating" varchar(2),
"dateCreated" timestamp(0) NOT NULL,
"dateUpdated" timestamp(0) NOT NULL,
"dateDeleted" timestamp(0) NULL DEFAULT NULL,
"uid" char(36) NOT NULL DEFAULT '0'
)
↵
Caused by: Database Exception – [yii\db\Exception](https://www.yiiframework.com/doc-2.0/yii-db-exception.html)

SQLSTATE[42804]: Datatype mismatch: 7 ERROR: column "green" is of type boolean but default expression is of type integer
HINT: You will need to rewrite or cast the expression.
The SQL being executed was: CREATE TABLE "carbontracker_stats" (
"id" serial NOT NULL PRIMARY KEY,
"entryId" integer NOT NULL,
"siteId" integer NOT NULL,
"url" varchar(255) NOT NULL,
"green" boolean DEFAULT 0,
"bytes" integer,
"cleanerThan" double precision DEFAULT 0,
"rating" varchar(2),
"dateCreated" timestamp(0) NOT NULL,
"dateUpdated" timestamp(0) NOT NULL,
"dateDeleted" timestamp(0) NULL DEFAULT NULL,
"uid" char(36) NOT NULL DEFAULT '0'
)

in /var/www/html/vendor/yiisoft/yii2/db/Schema.php at line 676
Error Info: Array
(
    [0] => 42804
    [1] => 7
    [2] => ERROR:  column "green" is of type boolean but default expression is of type integer
HINT:  You will need to rewrite or cast the expression.
)
↵
Caused by: PDOException

SQLSTATE[42804]: Datatype mismatch: 7 ERROR: column "green" is of type boolean but default expression is of type integer
HINT: You will need to rewrite or cast the expression.

in /var/www/html/vendor/yiisoft/yii2/db/Command.php at line 1302
```

Hope this is useful.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -2,7 +2,9 @@ name: ci
 on:
   workflow_dispatch:
   push:
-    branches: [ develop-v5 ]
+    branches:
+      - develop
+      - develop-v5
   pull_request:
 permissions:
   contents: read
@@ -15,6 +17,7 @@ jobs:
     uses: statikbe/.github/.github/workflows/ci.yml@main
     with:
       craft_version: '5'
+      php_version: '8.2'
       jobs: '["ecs", "phpstan"]'
     secrets:
       token: ${{ secrets.GITHUB_TOKEN }}

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -2,7 +2,7 @@ name: ci
 on:
   workflow_dispatch:
   push:
-    branches: [ develop ]
+    branches: [ develop-v5 ]
   pull_request:
 permissions:
   contents: read
@@ -14,7 +14,7 @@ jobs:
     name: ci
     uses: statikbe/.github/.github/workflows/ci.yml@main
     with:
-      craft_version: '4'
+      craft_version: '5'
       jobs: '["ecs", "phpstan"]'
     secrets:
       token: ${{ secrets.GITHUB_TOKEN }}

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Release Notes for carbon-tracker
 
+## 5.0.1 - 2024-04-10
+### Fixed
+- Fixed an installation error when using PostgreSQL ([#8](https://github.com/statikbe/craft-carbon-tracker/issues/8))
+
 ## 5.0.0 - 2024-03-12
 ### Added
 - Craft 5 support 🚀

--- a/composer.json
+++ b/composer.json
@@ -2,7 +2,7 @@
     "name": "statikbe/craft-carbon-tracker",
     "description": "Carbon tracker aims to raise awareness of the carbon emissions created by webpages, by displaying these insights along side the content in Craft's control panel.",
     "type": "craft-plugin",
-    "version": "5.0.0",
+    "version": "5.0.1",
     "license": "mit",
     "support": {
         "email": "support@statik.be",

--- a/src/migrations/Install.php
+++ b/src/migrations/Install.php
@@ -17,7 +17,7 @@ class Install extends Migration
                 'entryId' => $this->integer()->notNull(),
                 'siteId' => $this->integer()->notNull(),
                 'url' => $this->string()->notNull(),
-                'green' => $this->boolean()->defaultValue(0),
+                'green' => $this->boolean()->defaultValue(false),
                 'bytes' => $this->integer(),
                 'cleanerThan' => $this->float()->defaultValue(0.0),
                 'rating' => $this->string(2),

